### DISCUSSION TITLE
Firefox porting ?

### DISCUSSION THREAD

**Discussion by Ugau943** [Ademking/fb-profile-picture-viewer#13] [Ideas]
Hi, thanks for this great addon, and thus a solid work commitment, never disappointed !

Is there any plan or possibility to port it to Firefox ? Considering the interoperability (relative I know ...) of WebExtensions.

I don't think I've seen this topic discussed here before, so I'll take the liberty of doing so. Well, maybe it also means that there isn't much demand for it, but you never know.

**Comment by Ademking** [comment]
Thank you @Ugau943 for your feedback!

I will port this extension to Firefox very soon and I will tell you when It's ready 👍

**Reply by Ugau943** [reply]
As I said, solid work commitment. Thank you very much ...

**Comment by viczav** [comment]
I recently changed my main browser, and today I was wondering if there was a possibility of a Firefox port of this extension. I wanted to do it myself, but my knowledge of programming is very poor (almost zero, or practically none, lol). I guess I'll just keep on waiting, and thanks once again for your amazing work, Adem!

**Comment by DivanoDova** [comment]
Any update on this i also made the switch to firefox

**Comment by Reezlaw** [comment]
I have also switched to Firefox and I'd really like to bring this extension with me! Cheers
